runner/app/live/streamer/process.py

```diff
@@ -172,57 +172,86 @@ async def _initialize_pipeline(self):
                     raise
 
     async def _run_pipeline_loops(self):
-        pipeline = await self._initialize_pipeline()
-        input_task = asyncio.create_task(self._input_loop(pipeline))
-        output_task = asyncio.create_task(self._output_loop(pipeline))
-        param_task = asyncio.create_task(self._param_update_loop(pipeline))
+        self.pipeline = await self._initialize_pipeline()
```

Comment:
The main reason for not using fields instead of args is that it can be a little tricky. This class starts a new process so then it becomes two separate objects, one in the original class and one in the subprocess. It can be error-prone to rely on fields and then have things not really updated.

So if this is not a hard requirement, I'd say we should avoid fields as much as possible or consider creating a whole new class which is actually the "process state", which we only access from within the sub-process. WDYT?